src/emqx_connection.erl

```diff
@@ -629,10 +629,14 @@ parse_incoming(Data, Packets, State = #state{parse_state = ParseState}) ->
             NState = State#state{parse_state = NParseState},
             parse_incoming(Rest, [Packet|Packets], NState)
     catch
+        error:proxy_protocol_config_disabled:_Stk ->
+            ?LOG(error,
+                 "~nParse failed for proxy_protocol packet~nFrame data:~0p~n", [Data]),
```

Comment:
```suggestion
                 "~nMalfformed packets, please set "listener.NAME.ZONE.proxy_protocol" to true",
```
even better if `NAME` and `ZONE` is known here, then we can print out the exact config key like: `listener.tcp.external.proxy_protocol`

Reply:
Just LOG.
No `Stk` and `Packet Data`

Reply:
Yes. since we can be 100% sure it's a misconfiguration, the logs should tell the user exactly what's wrong.

Reply:
Seems doesn't have a good way to get `NAME` and `ZONE` of the current packet.
So just log as
`~nMalfformed packet, please check proxy_protocol config for current listener and zone~n`